packages/analytics-js/src/constants/logMessages.ts

```diff
@@ -84,6 +84,9 @@ const STORE_DATA_SAVE_ERROR = (key: string): string =>
 const STORE_DATA_FETCH_ERROR = (key: string): string =>
   `Failed to retrieve or parse data for "${key}" from storage`;
 
+const STORAGE_TYPE_VALIDATION_ERROR = (storageType: any): string =>
```

Comment:
Let's take `context` as input and format the message accordingly.

Reply:
Addressed